modules/core/src/main/scala/scala/build/errors/MultipleExcludeError.scala

```diff
@@ -0,0 +1,10 @@
+package scala.build.errors
+
+import scala.build.Position
+
+final class MultipleExcludeError(positions: Seq[Position], expectedProjectFilePath: os.Path)
```

Comment:
Nit: as it is used to mark exclude outside of `project.scala` I think it should be called `ExcludeDefinitionError`